src/core/ledger/ledger.test.js

```diff
@@ -784,6 +790,67 @@ describe("core/ledger/ledger", () => {
         expect(ledger.accountByName("target")).toEqual(null);
       });
     });
+
+    describe("setPayoutAddress", () => {
```

Comment:
Should also add a tests to ensure payoutAddresses are preserved after merging identities:
- identityA with address merged into identityB without address
- identityA without address merged into identityB with address
- identityA with address merged into identityB with address